pkg/kubeconfig/create_kubecfg_test.go

```diff
@@ -225,7 +225,7 @@ func TestBuildKubecfg(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := BuildKubecfg(tt.args.cluster, tt.args.keyStore, tt.args.secretStore, tt.args.status, tt.args.configAccess)
+			got, err := BuildKubecfg(tt.args.cluster, tt.args.keyStore, tt.args.secretStore, tt.args.status, tt.args.configAccess, true)
```

Comment:
Should add a test for `false`

Reply:
And fixed (which caught the above-mentioned error)